README.md

```diff
@@ -1 +1,11 @@
-# quoridor-react-native
\ No newline at end of file
+# Quoridor
+
+Quoridor client in React Native
+
```

Comment:
Missing install instructions

Reply:
It should be but it is now out of scope since the daily

Reply:
ok

Reply:
`install instructions` i mean. Ok for docker :+1: